test/TEST-03-JOBS/test-jobs.sh

```diff
@@ -10,44 +10,44 @@ while ! grep 'sleep\.service.*running' /root/list-jobs.txt; do
     systemctl list-jobs > /root/list-jobs.txt
 done
 
-grep 'hello\.service.*waiting' /root/list-jobs.txt || exit 1
+grep 'hello\.service.*waiting' /root/list-jobs.txt
 
 # This is supposed to finish quickly, not wait for sleep to finish.
 START_SEC=$(date -u '+%s')
 systemctl start --job-mode=ignore-dependencies hello
 END_SEC=$(date -u '+%s')
 ELAPSED=$(($END_SEC-$START_SEC))
 
-[ "$ELAPSED" -lt 3 ] || exit 1
+[ "$ELAPSED" -lt 3 ]
 
 # sleep should still be running, hello not.
 systemctl list-jobs > /root/list-jobs.txt
-grep 'sleep\.service.*running' /root/list-jobs.txt || exit 1
+grep 'sleep\.service.*running' /root/list-jobs.txt
 grep 'hello\.service' /root/list-jobs.txt && exit 1
-systemctl stop sleep.service hello-after-sleep.target || exit 1
+systemctl stop sleep.service hello-after-sleep.target
 
 # Test for a crash when enqueuing a JOB_NOP when other job already exists
-systemctl start --no-block hello-after-sleep.target || exit 1
+systemctl start --no-block hello-after-sleep.target
 # hello.service should still be waiting, so these try-restarts will collapse
 # into NOPs.
-systemctl try-restart --job-mode=fail hello.service || exit 1
-systemctl try-restart hello.service || exit 1
-systemctl stop hello.service sleep.service hello-after-sleep.target || exit 1
+systemctl try-restart --job-mode=fail hello.service
+systemctl try-restart hello.service
+systemctl stop hello.service sleep.service hello-after-sleep.target
 
 # TODO: add more job queueing/merging tests here.
 
 # Test for irreversible jobs
-systemctl start unstoppable.service || exit 1
+systemctl start unstoppable.service
 
 # This is expected to fail with 'job cancelled'
-systemctl stop unstoppable.service && exit 1
+systemctl stop unstoppable.service && exit 1 || true
```

Comment:
`A && B` seems to be sufficient here, so I don't think `A && B` should be replaced by `A && B || true`.